Misc/NEWS.d/next/Tests/2024-12-16-19-15-10.gh-issue-128003.GVBrfa.rst

```diff
@@ -0,0 +1,3 @@
+Add an option ``--parallel-threads=N`` to the regression test runner that
+runs individual tests in multiple threads in parallel in order to find
+concurrency bugs.
```

Comment:
Should this (or other documentation) mention that most of the testsuite isn't appropriately reviewed/marked yet?

Reply:
I've updated the blurb. It might be worth adding something to the devguide when this gets further along.